backend/src/main/resources/db/migration/V07__create_media.sql

```diff
@@ -0,0 +1,10 @@
+create table media(
+    ID serial primary key,
+    course_id integer references courses(ID),
+    order_index integer,
+    text varchar(1025),
```

Comment:
there is a text data type in postgres sql, use that and change this field name. Something like `content text` maybe